src/Common/Core/Impl/Shell/PDFExportOptions.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Common.Core {
+    public class PDFExportOptions {
+        public PDFExportOptions(string paperName, double paperWidth, double paperHeight) {
+            PaperName = paperName;
+            PaperWidth = paperWidth;
+            PaperHeight = paperHeight;
+        }
+
+        public static List<PDFExportOptions> GetPdfPapers(double pixelWidth, double pixelHeight) {
```

Comment:
IEnumerable or IReadOnlyCollection. Please do no return objects that can be modified unless it is necessary